tests/__init__.py

```diff
@@ -1,4 +1,10 @@
 import os
+import sys
+
+# FIXME: Search and replace these from the tests if pyarrow wheel is available
```

Comment:
This will get lost. How about a ticket? (Yeah, that one could be lost to).

Reply:
It won't, as I will come to this next week, as Python3.9 gets released.